OVERVIEW.md

```diff
@@ -0,0 +1,102 @@
+# Demo service overview
+
+## Architecture
+This service is a Django based service, running with a Celery message queue system. The compiled Docker image has two entry points to run; as either the Django based web frontend, or the Celery worker. The web service runs the web interface and webhook endpoints, and the Celery worker will perform any background tasks that have been added to the message queue.
+
+This service is usable in a variety of environments but it has been configured to run in a Rancher system. The rancher configuration is a `docker-compose.yml` file at its core, with some configuration options for Rancher. The service starts with these components:
+
+- `demoservice-lb`: Load balancer to route traffic to web services
+- App runs as two services
+  - `demoservice-web`: Django frontend service
+  - `demoservice-worker`: Celery worker
+- `demoservice-db`: Postgres database
+- `demoservice-rabbit`: RabbitMQ database
+
+Separately to this, a [Traefik](https://traefik.io/) instance runs to route traffic. This will be explained a little more clearly later in the document.
+
+
+### App structure
+
+A simple overview of the folder structure:
+
+```
+├── app
+│   └── demoservice
+│       ├── libs
+│       │   └── [DJANGO/PYTHON HELPER FILES]
+│       ├── migrations
+│       │   └── [DJANGO DATABASE MIGRATION SCRIPTS]
+│       └── templates
+│           └── [DJANGO HTML TEMPLATES]
+├── bin
+│   └── [HELPER SCRIPTS]
+└── templates
+    └── [RANCHER DEPLOYMENT TEMPLATE]
+```
+
+Inside the demoservice folder there are a couple of base files:
+- `tasks.py`
+  - Celery tasks and configuration
+- `views.py`
+  - webhook endpoint
+  - frontend management pages
+
+
+## Flow
+
+### Webhooks
+- Webhook, important to check secret
+ - Listens for pull_request events of type 'opened', 'closed', and 'synchronised'.
```

Comment:
You space doens't work on markdown, [check preview](https://github.com/canonical-webteam/demoservice/blob/af75dda7ea298b437f6b3aaf41f803136c26154a/OVERVIEW.md#webhooks)